Dockerfile

```diff
@@ -20,9 +20,10 @@ RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs \
 ENV PATH=${PATH}:/root/.cargo/bin
 
 # Add and build rush (synthetic network backend / userspace proxy)
-ADD rush /opt/lib/rush
-WORKDIR /opt/lib/rush
+ADD rush /opt/src/rush
+WORKDIR /opt/src/rush
```

Comment:
maybe I missed something, but curious why the move from `lib` to `src` here?

Reply:
The reason is that in the `minimal` docker image we build `rush` externally (outside the image) and then we copy the executable as `/opt/lib/rush` inside the image. In the regular image, we were copying the source code of rush to `/opt/lib/rush` (a directory in this case) and then we were calling `/opt/lib/rush/target/release/rush`.

So, to make it a little bit cleaner, for the regular image we now copy the source code to `/opt/src/rush` and once it's built then we copy the executable (now in `/opt/src/rush/target/release/rush`) to `/opt/lib/rush`. This way, both the regular image and the minimal image use the same executable in the same location.